lib/crutches/format/number.ex

```diff
@@ -102,7 +102,7 @@ defmodule Crutches.Format.Number do
   # Examples
 
       iex> Number.as_rounded(111.2345)
-      "111.235"
+      "111.234"
```

Comment:
Yeah I thought about this one a bit before changing it.  Python agrees with 1.4, ruby agrees with 1.3.  I'll change the test back.